sql/certifiedblocks/certifiedblocks.go

```diff
@@ -0,0 +1,61 @@
+package certifiedblocks
+
+import (
+	"fmt"
+	certtypes "github.com/spacemeshos/go-spacemesh/blockcerts/types"
+	"github.com/spacemeshos/go-spacemesh/codec"
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/sql"
+)
+
+func Add(db sql.Executor, blockCert *certtypes.BlockCertificate) error {
+    certBytes, err := codec.Encode(blockCert)
+    if err != nil {
+        return fmt.Errorf("encoding cert for block %s: %w",
+            blockCert.BlockID.String(), err)
+    }
+    _, err = db.Exec(
+        `insert into certified_blocks (block_id, layer_id, cert) 
```

Comment:
no need to associate layer_id to block cert. it should be 1-1 btwn blockID->cert